torch/jit/annotations.py

```diff
@@ -119,6 +120,11 @@ def get_type_line(source):
     type_lines = list(filter(lambda line: type_comment in line[1], lines))
 
     if len(type_lines) == 0:
+        type_pattern = re.compile('#[\t ]*type[\t ]*:')
+        wrong_type_lines = list(filter(lambda line: type_pattern.search(line[1]), lines))
+        if len(wrong_type_lines) > 0:
+            print("WARNING: the annotation prefix in line " + str(wrong_type_lines[0][0])
```

Comment:
This should either use the actual `warnings` module or be an error. I'm leaning towards it being an error since people should be fixing this if it comes up.

Reply:
let's turn this into an  error then.